python_modules/dagster/dagster_tests/cli_tests/command_tests/test_launch_command.py

```diff
@@ -236,27 +236,6 @@ def test_launch_using_memoization():
             assert len(run.step_keys_to_execute) == 0
 
 
-def test_job_launch_handles_pipeline():
-    job_kwargs = {
-        "workspace": None,
-        "job_name": "my_job",
-        "python_file": (file_relative_path(__file__, "repo_pipeline_and_job.py"),),
-        "module_name": None,
-        "attribute": "my_repo",
-    }
-    pipeline_kwargs = job_kwargs.copy()
-    pipeline_kwargs["job_name"] = "my_pipeline"
-
-    with default_cli_test_instance() as instance:
-        execute_launch_command(
-            instance,
-            job_kwargs,
-        )
-
-        # dont care if its a pipeline and not a job
-        execute_launch_command(instance, pipeline_kwargs)
```

Comment:
why is this being removed?

Reply:
Because the point of this test is to run the launcher on a legacy `PipelineDefinition`, which is being deleted.

Reply:
oof didn't read the test name lol.